test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow_custom_kfctl.sh

```diff
@@ -4,13 +4,13 @@ set -e
 install_kfctl(){
     #install kfctl cli
     KFCTL_VERSION="v1.0.2"
-    S3_BUCKET="kubeflow-kfctl-binary"
+    S3_URL="https://kubeflow-kfctl-binary.s3-us-west-2.amazonaws.com/kfctl_v1.0.2-1-g93e95e1_linux.tar.gz"
```

Comment:
nit: why is this s3 link regionalized and the kfctl hard coded?

Reply:
S3 bucket are regionalized. kfctl binary was provided to overcome https://github.com/kubeflow/kubeflow/issues/5245